fiftyone/operators/executor.py

```diff
@@ -830,6 +830,20 @@ def set_progress(self, progress=None, label=None):
         else:
             self.log(f"Progress: {progress} - {label}")
 
+    # TODO resolve circular import so this can have a type
```

Comment:
Understand the need to maintain consistency...but I think what is much more important is the developer experience for those building on top of the operator framework. Those poor souls don't want to consult the docs of ipython to see that a `value` can be of any type instead of a `str`.

Reply:
I think we should agree now to updating all of the modules you mentioned (operator modules) to have type hints for this reason ^